win32ss/user/ntuser/winpos.c

```diff
@@ -3103,6 +3105,83 @@ IntDeferWindowPos( HDWP hdwp,
     return retvalue;
 }
 
+/* Win: xxxImeWindowPosChanged */
+static VOID FASTCALL IntImeWindowPosChanged(PSMWP psmwp)
+{
+    PWND pwnd, pwndDesktop, pwndNode;
+    PTHREADINFO pti;
+    PWINDOWLIST pWL;
+    HWND hwnd, hImeWnd, *phwnd;
+    PIMEWND pImeWnd;
+    PIMEUI pimeui;
+
+    pwndDesktop = UserGetDesktopWindow();
+    if (!pwndDesktop)
+        return;
+
+    pti = gptiCurrent;
```

Comment:
Why not using gptiCurrent directly in the function?

Reply:
Fixed.